### PR TITLE
INC0001: remove unnecessary .dds files and improve/fix

### PR DESCRIPTION
**Pull request by TheKeyBlue**
Fixed the selection box and added the hover and rotate mechanic.
Still needs an image for the build tab.

I added the possibility to have the rotators at a specific angle or have them rotating based on what you set it to be in the blueprint 'Rotators' table.

Don't merge immediatly this time, review first.

### CODE REVIEW

File: units/INC0001/INC0001_unit.bp [hunk 1]
```diff
@@ -118,17 +125,29 @@ UnitBlueprint {
...
+        --if Stationary == false
+        PrimaryAccel = 5,
+        PrimarySpeed = 10,
+        SecondaryAccel = -5,
+        SecondarySpeed = -20,
+    },
+    SelectionSizeX = 3,
+    SelectionSizeZ = 8,
+    SelectionThickness = 0.15,
+    SizeX = 2.5,
```
Comment (speed2CZ) [inline units/INC0001/INC0001_unit.bp]: Arent there 3 values the hitbox?

**Comment by speed2CZ** [comment]
Maybe we should get some more sensible value for resource production as well, since this should be mostly T1 mission

**Comment by CookieNoob** [comment]
yea I would say 100e and 4mass?

File: units/INC0001/INC0001_unit.bp (hunk 1 above)
Comment (TheKeyBlue) [inline units/INC0001/INC0001_unit.bp]: euh no clue actually. (blueprint noob here)
I'll take a look at that tomorrow.

**Comment by JJ173** [comment]
If we have it generating resources, it needs to energy and mass storage too.

Each Nomad Engineer stores 10 mass but I found that there's no energy storage. That'll need changing too.

File: units/INC0001/INC0001_unit.bp (hunk 1 above)
Comment (speed2CZ) [inline units/INC0001/INC0001_unit.bp]: yes it is, and it's currently not efficient https://puu.sh/tsJRd/b64cd449ed.jpg
Shift F6 will open the window where you can edit the values, Ctrl Shift Alt C will show the hitbox
if you change it you need to spawn the unit again for the changes to show

**Comment by TheKeyBlue** [comment]
alright, i did the collision box, hoping its good.
And i put all the production/storage values to ACU values for now.

Don't know what to do with air/land being able to target it or not though.

**Comment by CookieNoob** [comment]
storage wont be very useful though... only the first player would profit from that. The others would still have no storage available. Better solution would be to place some mass/power storage outside of the playable region of the map.

**Comment by speed2CZ** [comment]
Offmap storage would be selectable, better way is to put into the script something like  `ArmyBrains[brain]:GiveStorage(type, number)` ;)

**Comment by CookieNoob** [comment]
so can this be merged?

**Comment by speed2CZ** [comment]
So currently it's hovering in the air, but it can be shot at only by tanks since its structure? That looks a bit bad.

Also 80k HP bit too much

**Comment by CookieNoob** [comment]
the problem is that inties have this in their weapon code:
            TargetRestrictDisallow = 'UNTARGETABLE,LAND,STRUCTURE,NAVAL',
            TargetRestrictOnlyAllow = 'AIR',

landunits dont have this... so as long as the unit is in the land layer, it should also be targetable by land units.

I suggest to remove the Tags "Structure", "Land". This will make it targetable by inties and landunits.

15k hp would be better since we encounter mostly t1 during the mission.

**Comment by TheKeyBlue** [comment]
changed the categories, so targetable by air and land, but not maa or static aa.

**Comment by TheKeyBlue** [comment]
ok changed hp and elevation.

Was trying to get it to land after getting destroyed, but that didn't work out as i had hoped.
Basicly wasted multiple hours on that. :(

Mainly cause of how the targetting is done, no clue on what shoots what and why.

**Pull request merged**